packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -0,0 +1,450 @@
+
+```tsx example
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Menu Controlled
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    selectionMode="single"
+    selectedKeys={['Kangaroo']}>
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Menu Uncontrolled
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    selectionMode="single"
+    defaultSelectedKeys={['Snake']}>
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+The `selectionMode` prop can be used to switch from the default selection of a single Menu Item to multiple Menu Items or none.
+
+```tsx example
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Menu Selection Multiple
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    defaultSelectedKeys={['Kangaroo', 'Snake']}
+    selectionMode="multiple">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Menu Selection None
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    selectionMode="none">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+### Sections
+
+Menus may have Sections with titles and children items.
+
+```tsx example
+<MenuTrigger>
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Section 1', children: [{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}, {name: 'Section 2', children: [{name: 'Koala'}]}]} itemKey="name">
+    {item => (
+      <Section items={item.children} title={item.name}>
+        {item => <Item>{item.name}</Item>}
+      </Section>
+    )}
+  </Menu>
+</MenuTrigger>
+```
+
+### Sematic Elements
+
+A Menu item's content may be any renderable node, not just strings.
+
+[View guidelines](https://spectrum.adobe.com/page/popovers/#Content-types)
+
+```tsx example
+import {Keyboard, Text} from '@react-spectrum/typography';
+
+<MenuTrigger>
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu
+    itemKey="name"
+    items={[
+      {name: 'Copy', icon: 'Copy', shortcut: '⌘C'},
+      {name: 'Cut', icon: 'Cut', shortcut: '⌘X'},
+      {name: 'Paste', icon: 'Paste', shortcut: '⌘V'}
+    ]}>
+    {item => {
+      let iconMap = {
+        Copy,
+        Cut,
+        Paste
+      };
+      let Icon = iconMap[item.icon];
+      return (
+        <Item childItems={item.children} textValue={item.name}>
+          <Icon size="S" />
+          <Text>{item.name}</Text>
+          <Keyboard>{item.shortcut}</Keyboard>
+        </Item>
+      );
+    }}
+  </Menu>
+</MenuTrigger>
+```
+
+### Accessibility
+
+Menu sections without labels use the prop `aria-label`.
```

Comment:
maybe "Titleless Menu sections must be provided with an `aria-label` for accessibility"? Brings it more inline with how we usually described `aria-label`